docs/snippets/angular/page-story-slots.ts.mdx

```diff
@@ -5,26 +5,27 @@ import type { Meta, StoryObj } from '@storybook/angular';
 
 import { Page } from './page.component';
 
-const meta: Meta<Page> = {
-  component: Page,
-};
-
-export default meta;
-type Story = StoryObj<Page>;
+// TODO: Is this correct?
+//       If not, is it expected that an Angular component exports its props type?
+//       What if it's an interface instead of a type (which seems to be the convention)?
+//       Or, how do you extract an Angular component's props type?
+type PagePropsAndCustomArgs = Page & { footer?: string };
```

Comment:
What do you mean with this interface convention? I haven't heard of that.

I think the component will always contain all props (and more), so it kind of works, but users might get autocompletion for args that don't really exist.

I guess that is an acceptable tradeoff? For people that want really strict types, they would need to do something like:
`type Story = Pick<Page, 'prop1' | 'prop2'>`

I don't think the utility you want exists, I also asked it here:
https://github.com/storybookjs/storybook/discussions/20988